lib/APPLineFollowerSensorFusion/src/App.cpp

```diff
@@ -84,21 +86,37 @@ const char* App::TOPIC_NAME_WILL = "will";
 /** MQTT topic name for status messages. */
 const char* App::TOPIC_NAME_STATUS = "zumo/status";
 
-/** MQTT topic name for fusion Pose */
+/** MQTT topic name for fusion pose. */
 const char* App::TOPIC_NAME_FUSION_POSE = "zumo/fusion";
 
-/** MQTT topic name for raw Sensor data */
+/** MQTT topic name for raw sensor data. */
 const char* App::TOPIC_NAME_RAW_SENSORS = "zumo/sensors";
 
-/** MQTT topic name for receiving Space Ship Radar Pose. */
+/** MQTT topic name for receiving Space Ship Radar pose. */
 const char* App::TOPIC_NAME_RADAR_POSE = "ssr";
 
-/** Buffer size for JSON serialization of birth / will message */
+/** MQTT topic name for host time sync request. */
+const char* App::TOPIC_NAME_HOST_TIMESYNC_REQ = "zumo/time_sync/request";
+
+/** MQTT topic name for host time sync response. */
+const char* App::TOPIC_NAME_HOST_TIMESYNC_RSP = "zumo/time_sync/response";
+
+/** Buffer size for JSON serialization of birth / will message. */
 static const uint32_t JSON_BIRTHMESSAGE_MAX_SIZE = 64U;
 
-/** Buffer size for JSON serialization of combined sensor snapshot */
+/** Buffer size for JSON serialization of combined sensor snapshot. */
 static const uint32_t JSON_SENSOR_SNAPSHOT_MAX_SIZE = 256U;
 
+/** Buffer size for JSON serialization of fusion pose. */
+static const uint32_t JSON_FUSION_POSE_MAX_SIZE = 128U;
+
+/* Convenience aliases for EKF types. */
+using ImuMeasVector = ExtendedKalmanFilter5D::ImuMeasVector;
```

Comment:
Please write the long forms of the words (Measurement instead of Meas) in order to improve readability